crates/sui-framework/packages/sui-system/sources/staking_pool.move

```diff
@@ -676,6 +687,42 @@ module sui_system::staking_pool {
     #[test_only]
     public use fun fungible_staked_sui_data_principal_value as FungibleStakedSuiData.principal_value;
 
+    // NEVER remove `#[test_only]`
```

Comment:
I'm not so sure about the need to add this comment, given that it's a `_test_only` function, and even if it wasn't, visibility will not expose it further.

Reply:
will change to use `for_testing` suffix
for the comment, I followed this: https://github.com/MystenLabs/sui/blob/main/crates/sui-framework/packages/sui-system/sources/sui_system.move#L744 as it's "write" function that will pollute system data, making them more dangerous than test-only "read" functions. But now i realized i accidentally added the same comments to new "read" test only functions too.

To me this is just a hygiene thing. Do you want me to remove them entirely or just "read" functions?

Reply:
> for the comment, I followed this: https://github.com/MystenLabs/sui/blob/main/crates/sui-framework/packages/sui-system/sources/sui_system.move#L744 as it's "write" function that will pollute system data, making them more dangerous than test-only "read" functions. But now i realized i accidentally added the same comments to new "read" test only functions too.

Not a fan of this comment either. 😕 
I think in our practice having `_for_testing` and `#[test_only]` annotation is more than enough. Adding comments seems redundant.

Reply:
fixed